src/odemis/gui/win/acquisition.py

```diff
@@ -612,6 +612,409 @@ def on_acquisition_done(self, future):
+        self._view.mpp.value = orig_view.mpp.value
+        self._view.merge_ratio.value = orig_view.merge_ratio.value
+
+        # attach the view to the viewport
+        self.pnl_view_acq.canvas.fit_view_to_next_image = False
+        self.pnl_view_acq.setView(self._view, self._tab_data_model)
+
+        # The TOOL_ROA is not present because we don't allow the user to change
+        # the ROA), so we need to explicitly request the canvas to show the ROA.
+        if hasattr(self._tab_data_model, "roa") and self._tab_data_model.roa is not None:
+            cnvs = self.pnl_view_acq.canvas
+            self.roa_overlay = RepetitionSelectOverlay(cnvs, self._tab_data_model.roa,
+                                                             self._tab_data_model.fovComp)
+            cnvs.add_world_overlay(self.roa_overlay)
+
+        self.Bind(wx.EVT_CHAR_HOOK, self.on_key)
+
+        self.btn_cancel.Bind(wx.EVT_BUTTON, self.on_close)
+        self.btn_secom_acquire.Bind(wx.EVT_BUTTON, self.on_acquire)
+        self.Bind(wx.EVT_CLOSE, self.on_close)
+        # on_streams_changed is compatible because it doesn't use the args
+
+        # To update the estimated time when streams are removed/added
+        self._view.stream_tree.flat.subscribe(self.on_streams_changed)
+        self._hidden_view.stream_tree.flat.subscribe(self.on_streams_changed)
+
+    def start_listening_to_va(self):
+        # Get all the VA's from the stream and subscribe to them for changes.
+        for entry in self._orig_entries:
+            if hasattr(entry, "vigilattr"):
+                entry.vigilattr.subscribe(self.on_setting_change)
+
+    def stop_listening_to_va(self):
+        for entry in self._orig_entries:
+            if hasattr(entry, "vigilattr"):
+                entry.vigilattr.unsubscribe(self.on_setting_change)
+
+    def duplicate_tab_data_model(self, orig):
+        """
+        Duplicate a MicroscopyGUIData and adapt it for the acquisition window
+        The streams will be shared, but not the views
+        orig (MicroscopyGUIData)
+        return (MicroscopyGUIData)
+        """
+        # TODO: we'd better create a new view and copy the streams
+        new = copy.copy(orig)  # shallow copy
+
+        new.streams = model.ListVA(orig.streams.value)  # duplicate
+
+        # create view (which cannot move or focus)
+        view = guimodel.MicroscopeView("All")
+
+        # differentiate it (only one view)
+        new.views = model.ListVA()
+        new.views.value.append(view)
+        new.focussedView = model.VigilantAttribute(view)
+        new.viewLayout = model.IntEnumerated(guimodel.VIEW_LAYOUT_ONE,
+                                             choices={guimodel.VIEW_LAYOUT_ONE})
+        new.tool = model.IntEnumerated(TOOL_NONE, choices={TOOL_NONE})
+        return new
+
+    def add_all_streams(self):
+        """
+        Add all the streams present in the interface model to the stream panel.
+        """
```

Comment:
It should only add the live streams. In particular, the static streams shouldn't be added. For instance, if the user acquires the overviews, and want to acquire them again, the previous overviews shouldn't appear in this window.

Actually, it might be even "safer" to explicitly return only one SEM stream, and the FluoStreams.